proto/spire/api/server/logger/v1/logger.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package spire.api.server.logger.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/server/logger/v1;loggerv1";
+
+import "spire/api/types/logger.proto";
+
+service Logger {
+
+    // Gets the logger level.
+    //
+    // The caller must be local or present an admin X509-SVID.
+    rpc GetLogger(GetLoggerRequest) returns (spire.api.types.Logger);
+
+    // Sets the logger to a specified log level.
+    //
+    // The caller must be local or present an admin X509-SVID.
+    rpc SetLogLevel(SetLogLevelRequest) returns (spire.api.types.Logger);
+
+    // Resets the logger level to the level configured at launch.
+    //
+    // The caller must be local or present an admin X509-SVID.
+    rpc ResetLogLevel(ResetLogLevelRequest) returns (spire.api.types.Logger);
+
+}
+
+// Empty Get Logger Request :essage for future extension
+message GetLoggerRequest {
+}
+
+// The requested log level the Logger should assume.
+message SetLogLevelRequest {
+
+    // The new level the logger should assume
+    spire.api.types.LogLevel set_level = 1;
```

Comment:
```suggestion
    spire.api.types.LogLevel log_level = 1;
```

Reply:
I'd rather keep it as "set_level" as "log_level" can confuse input / output orientation of the value.

If set_level seems wrong, anything indicating a request would be fine by me.  I just don't want the stammering of saying it's a log level type and then having nothing else to say except it's a log level (which is captured by the type).